src/txn/txn.c

```diff
@@ -2789,9 +2795,13 @@ __wt_verbose_dump_txn_one(
 {
     WT_TXN *txn;
     WT_TXN_SHARED *txn_shared;
-    char buf[512];
+    uint32_t i, buf_len;
     char ts_string[6][WT_TS_INT_STRING_SIZE];
     const char *iso_tag;
+    char snapshot_buf_tmp[32];
+    WT_DECL_ITEM(buf);
+    WT_DECL_ITEM(snapshot_buf);
+    WT_DECL_RET;
```

Comment:
These declarations need to move above WT_TXN to be alphabetical.

Reply:
I have adjusted the position,thanks.